src/specify_cli/__init__.py

```diff
@@ -58,6 +58,7 @@
     "claude": "Claude Code",
     "gemini": "Gemini CLI",
     "cursor": "Cursor",
+    "kilocode": "Kilo Code",
```

Comment:
Any changes to `__init__.py` should be accompanied by a package version rev and update to the `CHANGELOG.md`